### PR TITLE
docs: changelog for 5.20.6

### PR DESCRIPTION
**Pull request by PeachScript**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 📝 Change Log

- Dropdown
  - 🐞 Fix Dropdown cannot wrap correctly when out of screen edge. [#50752](https://github.com/ant-design/ant-design/pull/50752) [@afc163](https://github.com/afc163)
  - 🐞 Fix Dropdown cannot wrap correctly. [#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
- 🐞 Improve Menu collapse animation smoothness. [#50751](https://github.com/ant-design/ant-design/pull/50751) [@afc163](https://github.com/afc163)
- 🐞 Fix Table cell overflow bug if edit with virtual scroll. [#50737](https://github.com/ant-design/ant-design/pull/50737) [@huiliangShen](https://github.com/huiliangShen)
- 🐞 Fix Input.Search button radius not changing with `size`. [#50734](https://github.com/ant-design/ant-design/pull/50734) [@afc163](https://github.com/afc163)
- 🐞 Fix Form password still can be toggle show/hide even if disabled. [#50616](https://github.com/ant-design/ant-design/pull/50616) [@Jarryxin](https://github.com/Jarryxin)
- 💄 Fix Badge background transition when mouse out. [#50743](https://github.com/ant-design/ant-design/pull/50743) [@coding-ice](https://github.com/coding-ice)
- TypeScript
  - 🤖 Fix Collapse types for `onChange` arguments. [#50754](https://github.com/ant-design/ant-design/pull/50754) [@yuanliu147](https://github.com/yuanliu147)

---

- Dropdown
  - 🐞 修复 Dropdown 菜单项在屏幕视口外时未能正确换行的问题。[#50752](https://github.com/ant-design/ant-design/pull/50752) [@afc163](https://github.com/afc163)
  - 🐞 修复 Dropdown 菜单项未能正确换行的问题。[#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
- 🐞 修复 Menu 折叠动画不够丝滑的问题。[#50751](https://github.com/ant-design/ant-design/pull/50751) [@afc163](https://github.com/afc163)
- 🐞 修复 Table 虚拟滚动时单元格宽度可能溢出的问题。[#50737](https://github.com/ant-design/ant-design/pull/50737) [@huiliangShen](https://github.com/huiliangShen)
- 🐞 修复 Input.Search 的按钮圆角不随 `size` 变化的问题。[#50734](https://github.com/ant-design/ant-design/pull/50734) [@afc163](https://github.com/afc163)
- 🐞 修复 Form 禁用时仍可切换密码显隐的问题。[#50616](https://github.com/ant-design/ant-design/pull/50616) [@Jarryxin](https://github.com/Jarryxin)
- 💄 修复 Badge 鼠标移出时无背景色过渡动画的问题。[#50743](https://github.com/ant-design/ant-design/pull/50743) [@coding-ice](https://github.com/coding-ice)
- TypeScript
  - 🤖 修复 Collapse 的 `onChange` 的函数入参类型。[#50754](https://github.com/ant-design/ant-design/pull/50754) [@yuanliu147](https://github.com/yuanliu147)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ tag: vVERSION
...
+
+- Dropdown
+  - 🐞 Fix Dropdown cannot wrap correctly when out of screen edge. [#50752](https://github.com/ant-design/ant-design/pull/50752) [@afc163](https://github.com/afc163)
+  - 🐞 Fix Dropdown cannot wrap correctly. [#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
+- 🐞 Improve Menu collapse animation smoothness. [#50751](https://github.com/ant-design/ant-design/pull/50751) [@afc163](https://github.com/afc163)
+- 🐞 Fix Table cell overflow bug if edit with virtual scroll. [#50737](https://github.com/ant-design/ant-design/pull/50737) [@huiliangShen](https://github.com/huiliangShen)
+- 🐞 Fix Input.Search button radius not changing with `size`. [#50734](https://github.com/ant-design/ant-design/pull/50734) [@afc163](https://github.com/afc163)
+- 🐞 Fix Form password still can be toggle show/hide even if disabled. [#50616](https://github.com/ant-design/ant-design/pull/50616) [@Jarryxin](https://github.com/Jarryxin)
+- 💄 Fix Badge background transition when mouse out. [#50743](https://github.com/ant-design/ant-design/pull/50743) [@coding-ice](https://github.com/coding-ice)
+
```
Comment (afc163) [inline CHANGELOG.en-US.md:31]: 多余空行

**Comment by crazyair** [comment]
https://github.com/ant-design/ant-design/pull/50695 这个没有

**Comment by PeachScript** [comment]
> #50695 这个没有

这个上周一 @zombieJ 发的 patch 日志里已包含：https://github.com/ant-design/ant-design/releases/tag/5.20.5

**Comment by crazyair** [comment]
> > #50695 这个没有
> 
> 这个上周一 @zombieJ 发的 patch 日志里已包含：https://github.com/ant-design/ant-design/releases/tag/5.20.5

一共 2 个点，上次只写了一个点

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ tag: vVERSION
 
 ---
 
+## 5.20.6
+
+`2024-09-09`
+
+- Dropdown
+  - 🐞 修复 Dropdown 菜单项在屏幕视口外时未能正确换行的问题。[#50752](https://github.com/ant-design/ant-design/pull/50752) [@afc163](https://github.com/afc163)
+  - 🐞 修复 Dropdown 菜单项未能正确换行的问题。[#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
```
Comment (zombieJ) [inline CHANGELOG.zh-CN.md:24]: 这两条内容差不多，log 合并然后放两个 PR ？

Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🐞 回滚 [#49899](https://github.com/ant-design/ant-design/pull/49899) 以修复 Dropdown 菜单项文字溢出菜单的问题，并重新修复屏幕视口外菜单项内容换行错误的问题。[#50752](https://github.com/ant-design/ant-design/pull/50752) [#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,19 @@ tag: vVERSION
 
 ---
 
+## 5.20.6
+
+`2024-09-09`
+
+- 🐞 修复 Dropdown 菜单项在某些情况下未能正确换行的问题。[#50752](https://github.com/ant-design/ant-design/pull/50752) [#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:22]:
```suggestion
- 🐞 回滚 [#49899](https://github.com/ant-design/ant-design/pull/49899) 以修复 Dropdown 菜单项文字溢出菜单的问题，并重新修复屏幕视口外菜单项内容换行错误的问题。[#50752](https://github.com/ant-design/ant-design/pull/50752) [#50718](https://github.com/ant-design/ant-design/pull/50718) [@afc163](https://github.com/afc163)
```

**Comment by zombieJ** [comment]
> 一共 2 个点，上次只写了一个点

单独来个 PR 改上个 Patch 的 changelog 吧。这个变更不是这次带上的。

**Review by afc163 — APPROVED** [review]

**Pull request merged**